### DISCUSSION TITLE
Adding a *className* attribute to react native components doesn't make vs code happy

### DISCUSSION THREAD

**Discussion by regisrex** [nativewind/nativewind#280] [Q&A]
Hello World! I've been trying to use native wind in my react native (with expo) project to make styling quick but when I add  the <code>className </code>  attribute to  a react native element, vs code marks an error , but when previewing my app,  the styles work well.  <br>
![image](https://user-images.githubusercontent.com/94565752/194695952-23acd9fb-4c04-427e-80b1-7adcc5a7e95e.png)

Anyone with an idea  how  i can configure my preject to stop showing me error on the <code>className</code> attribute ?

**Comment by steida** [comment]
USe https://www.nativewind.dev/getting-started/typescript, Luke ✨

**Comment by johhansantana** [comment]
Where you able to fix this? I'm getting the same error and creating the file with the content provided in the documentation doesn't seem to fix it.

**EDIT**
Nevermind, my IDE was giving me trouble by excluding `.d.ts` files for some reason. Had to `rm -rf .idea` and restart IDE and it worked.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
